pkg/controller/resourcequota/resource_quota_controller.go

```diff
@@ -155,20 +163,21 @@ func NewController(options *ControllerOptions) (*Controller, error) {
 			replenishmentFunc: rq.replenishQuota,
 			registry:          rq.registry,
 			updateFilter:      options.UpdateFilter,
+			clusterName:       options.ClusterName,
 		}
 
 		rq.quotaMonitor = qm
 
 		// do initial quota monitor setup.  If we have a discovery failure here, it's ok. We'll discover more resources when a later sync happens.
 		resources, err := GetQuotableResources(options.DiscoveryFunc)
 		if discovery.IsGroupDiscoveryFailedError(err) {
-			utilruntime.HandleError(fmt.Errorf("initial discovery check failure, continuing and counting on future sync update: %v", err))
+			utilruntime.HandleError(fmt.Errorf("%s: initial discovery check failure, continuing and counting on future sync update: %v", options.ClusterName, err))
```

Comment:
contextual logging will make this unneeded, right? But we don't use it yet in HandleError I guess.

Reply:
Probably correct on both accounts. However, in 1.24, the resource quota controller hasn't been converted to use contextual logging.